message_definitions/v1.0/ras_a.xml

```diff
@@ -245,6 +256,12 @@
         <description>Cylinder.</description>
       </entry>
     </enum>
+    <enum name="MAV_MISSION_TYPE">
+      <description>Type of mission items being requested/sent in mission protocol.</description>
+      <entry value="3" name="MAV_MISSION_TYPE_RADIO">
+        <description>Specifies the channel switching frequencies. Items are MAV_CMD_DO_SET_RF_ RadioFrequency items.</description>
+      </entry>
+    </enum>
```

Comment:
so i guess adding this here, is what brought the entire enum into the ras_a.h, interesting. i think thats fine :/